__tests__/comments.test.js

```diff
@@ -0,0 +1,64 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/articles/:article_id/comments', () => {
+    let validArticleId;
+
+
+    beforeEach(() => {
+        return db.query('SELECT article_id FROM articles LIMIT 1;').then(({ rows }) => {
+            validArticleId = rows[0].article_id;
+        });
+    });
+
+    test('200: responds with an array of comment objects for the given article_id, sorted by date in descending order', () => {
+        return request(app)
+        .get(`/api/articles/${validArticleId}/comments`)
```

Comment:
lets test using a article_id that we know exists as we can have a look at our test data.